ru.lproj/Localizable.strings

```diff
@@ -0,0 +1,160 @@
+/*
+  Localizable.strings
+  MobileRadio
+
+  Created by Andrei Vidrasco on 10/05/2019.
+  Copyright © 2019 High Caffeine Content. All rights reserved.
+*/
+
+"SHORTCUT_PLAY" = "Воспроизвести";
+"SHORTCUT_PAUSE" = "Пауза";
+
+"TOOLBAR_NOTHING_PLAYING" = "Ничего не играет";
+"TOOLBAR_STREAMING_RADIO" = "Потоковое Радио";
+
+"TOOLBAR_PLAYBUTTON_ACCESSIBILITY_LABEL" = "Воспроизвести/Пауза";
+"TOOLBAR_PLAYBUTTON_TOOLTIP" = "Воспроизвести/Пауза";
+"TOOLBAR_MUTEBUTTON_TOOLTIP" = "Переключить звук";
```

Comment:
Yeah, but then in case sound is already off, it would still write "Turn off sound". Not sure it's the best idea. Steven, what do you think?

Reply:
As it stands in the app right now, the tooltip is used for mousing over both states of the mute button — when sound is enabled, and when sound is disabled. How would that change the translation?

<img width="144" alt="Screen Shot 2020-05-10 at 17 02 56" src="https://user-images.githubusercontent.com/45212/81504245-1e7c1780-92e0-11ea-882d-4541d445c94c.png">

Reply:
> Not sure it's the best idea.
I agree with you, we need to find a better solution. The problem is that «переключить звук» is confusing because it usually means ‘change sound output’ (e.g. «переключить звук на наушники» = ‘change sound output to headphones’). What if we translate it as «Отключить/включить звук» (‘turn off/on sound’)?

Reply:
We can. But I check different values in the system itself, and "переключить звук" is acceptable translation as well. @steventroughtonsmith in case you need it for future references, apple uses "toggle" in mail app, when you press drop-down on a flag
<img width="162" alt="Screenshot 2020-05-10 at 18 44 30" src="https://user-images.githubusercontent.com/4473783/81505292-63a74600-92ee-11ea-958f-e32f27e5b7e5.png">

so translation for this in russian is 
<img width="220" alt="Снимок экрана 2020-05-10 в 18 36 52" src="https://user-images.githubusercontent.com/4473783/81505309-820d4180-92ee-11ea-898c-2107957cdd64.png">
So, I think it's completely acceptable (and on par with apple's system) translation, but of course, we can use more descriptive term of "turn off/on sound".
@steventroughtonsmith your decision

Reply:
On second thought, Robert was making different point.
It would be acceptable translation in case in russian, there would be a word for "mute" in this context. We're just using "sound", so toggle sound, might bring confusion. Let's go with descriptive term, "turn off/on"